buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/fixture/hadoop/tasks/HiveBeeline.groovy

```diff
@@ -23,15 +23,19 @@ import org.elasticsearch.hadoop.gradle.fixture.hadoop.conf.HadoopClusterConfigur
 import org.elasticsearch.hadoop.gradle.fixture.hadoop.conf.InstanceConfiguration
 import org.elasticsearch.hadoop.gradle.fixture.hadoop.services.HiveServiceDescriptor
 import org.gradle.api.GradleException
+import org.gradle.api.tasks.Internal
 import org.gradle.api.tasks.TaskAction
 import org.gradle.process.ExecSpec
 
 import static org.elasticsearch.hadoop.gradle.fixture.hadoop.conf.SettingsContainer.FileSettings
 
 class HiveBeeline extends AbstractClusterTask {
 
+    @Internal
```

Comment:
Same re: `@Input`

We should probably change this file type to a path type since I saw that Gradle is weird about File types for Inputs

Reply:
Have a reference for the weirdness? This seems to make it sound like it's OK as long as it's not a relative File, right? https://docs.gradle.org/current/userguide/working_with_files.html#sec:locating_files

Reply:
I based it off of the `@Input` annotation docs. https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/Input.html

Reply:
Ah I see you're using `@InputFile` instead. All good!